app/src/pages/inside/stepPage/stepGrid/defectType/patternAnalyzedLabel/patternAnalyzedLabel.jsx

```diff
@@ -0,0 +1,23 @@
+import classNames from 'classnames/bind';
+import PropTypes from 'prop-types';
+import { withHoverableTooltip } from 'components/main/tooltips/hoverableTooltip';
+import { PatternAnalyzedTooltip } from './patternAnalyzedTooltip';
+import styles from './patternAnalyzedLabel.scss';
+
+const cx = classNames.bind(styles);
+
+export const PatternAnalyzedLabel = withHoverableTooltip({
```

Comment:
Should this tooltip be hoverable? I don't see any action in the body of the tooltip

Reply:
Fixed.